dxr/plugins/xpidl/__init__.py

```diff
@@ -0,0 +1,28 @@
+from functools import partial
```

Comment:
Mark this day on your calendar: I'm going to recommend you do something like the clang plugin. ;-) I could really use a little docstring here describing the general approach to analyzing IDL.